tests/unit/test_zowe_core.py

```diff
@@ -380,6 +384,27 @@ def test_profile_not_found_warning(self, get_pass_func):
             prof_manager = ProfileManager()
             prof_manager.config_dir = self.custom_dir
             props: dict = prof_manager.load("non_existent_profile")
+    
+    @patch("keyring.get_password", side_effect=keyring_get_password)
+    def test_profile_loading_with_valid_schema(self, get_pass_func):
+        """
+        Test Validation, no error should be raised for valid schema
+        """
+        # Setup - copy profile to fake filesystem created by pyfakefs
+        custom_file_path = os.path.join(self.custom_dir, "zowe.config.json")
+        shutil.copy(self.original_nested_file_path, custom_file_path)
+        custom_file_path = os.path.join(self.custom_dir, "zowe.schema.json")
```

Comment:
@t1m0thyj I have added the test but the fake file system is not taking the `zowe.schema.json` file